web/modules/custom/bglobal_angular/components/voices_list/template.html

```diff
@@ -1,6 +1,13 @@
-<div *ngFor="let post of posts" data-ng-cloak class="voices-section grid-flip" style="width: 400px; padding: 10px;">
-  <!-- {{ i }} {{ getKeys(post) }} -->
-  <img [src] = "post.image.src" [alt] = "post.image.alt" style="width: 400px;" class="voice-section--img m-b-20 img-fluid" />
-  <p class="marginless-p"><strong><a [href] = "post.path" class="blog-header">{{ post.title }}</a></strong></p>
-  <div class="blog-intro" [innerHTML] = "post.content"></div>
-</div>
+<div class="voices">
+  <h2 class="block-header">Latest</h2>
```

Comment:
Make sure the block is configured to hide the title if you are going to code the title here. 

Given that we may use this block in multiple places, is `Latest` always going to be the label, or do we need that to be a configuration option?

Reply:
For now I believe we'll always want it to say latest.

Reply:
`Make sure the block is configured to hide the title if you are going to code the title here.`
@agentrickard I assume this is configurable in the info.yml? Is there documentation on what configurable keys are available to use?

Reply:
It is not configurable in the YML. It is a core setting for every block. When you assign this block to a region, you will need to disable the title each time.